synonymdev/bitkit-react-native
# chore(wallet): Upgrade react-native to 0.71.7

# Description

- Upgrades `react-native` to `0.71.7`.
- Upgrades dependencies in `package.json`.
- Android is now using `hermes` instead of the `v8` engine.

## Type of change
- [x] Dependency Upgrade (non-breaking changes which adds functionality)

## QA Notes

- Prior to running Bitkit we need to first specify either `yarn setup-ios` or `yarn setup-android`.
    - This will be removed once we have a stable `libsodium` branch for both iOS & Android.
- Please suggest changes or create PR's against this branch if you see anything that needs fixing or additional attention.
- All functionality throughout the app should be tested to ensure that functionality is retained or improved.
